src/compas_viewer/__main__.py

```diff
@@ -0,0 +1,57 @@
+# ==========================================================================
+# python -m compas_viewer -f "file path or directory path"
+# ==========================================================================
+
+import argparse
+from os import listdir
+from os import path
+
+from compas import json_load
+from compas import json_loadz
+
+from compas_viewer import Viewer
+
+ap = argparse.ArgumentParser()
+ap.add_argument(
+    "-f",
+    "--files",
+    required=False,
+    help="""
+    The compas.geometry's JSON file, or the compressed JSON file (ZIP),
+    or a the path to a folder containing the JSON files for the viewer to load.""",
+)
+
+args = vars(ap.parse_args())
+
+# ==========================================================================
+# "-f" argument
+# ==========================================================================
+
+geometries = []
+if args["files"]:
+    if path.isdir(args["files"]):
+        file_names = listdir(args["files"])
+        for file_name in file_names:
+            if file_name.endswith(".json"):
+                geometries.append(json_load(path.join(args["files"], file_name)))
+
+    elif args["files"].endswith(".json"):
+        geometries.append(json_load(args["files"]))
```

Comment:
there needs to be a detection mechanism here before adding them to a list of geometries, because the content of json can take many possible shapes, for example:
`{geometry}`
or
`[{geometryA}, {geometryB}]`
or mixed with non-geo objects:
`[[{geometryA}, {geometryB}, {non-geo C}]`

Reply:
added `Validate geometries` section